test/lightning_web/live/work_order_live_test.exs

```diff
@@ -1,13 +1,18 @@
 defmodule LightningWeb.RunWorkOrderTest do
-  alias Lightning.Attempts
-  use LightningWeb.ConnCase, async: true
```

Comment:
Ah man, why do we have to go synchronous?

Reply:
Some tests get flaky. I have turned off in the middle of the tests and trying again now got this once:
```
1) test Search and Filtering bulk select isn't available when there's no workorder matching the filter (LightningWeb.RunWorkOrderTest)
     test/lightning_web/live/work_order_live_test.exs:637
     Expected truthy, got false
     code: assert has_element?(view, "#select_all")
     arguments:

         # 1
         #Phoenix.LiveViewTest.View<id: "phx-F5n67eMZN-U4jUNB", module: LightningWeb.RunLive.Index, pid: #PID<0.1288.0>, endpoint: LightningWeb.Endpoint, ...>

         # 2
         "#select_all"

     stacktrace:
       test/lightning_web/live/work_order_live_test.exs:684: (test)
``` 
Only with `async: true`

It's a difference of only from 3.3 to 3.7 seconds